.github/workflows/java-eks-e2e-test.yml

```diff
@@ -358,7 +362,7 @@ jobs:
           --service-name sample-application-${{ env.TESTING_ID }}
           --remote-service-deployment-name ${{ env.REMOTE_SERVICE_DEPLOYMENT_NAME }}
           --remote-resource-identifier "information_schema|${{env.RDS_MYSQL_CLUSTER_ENDPOINT}}|3306"
-          --remote-db-user "admin"
+          --remote-db-user ${{ env.REMOTE_DB_USER }}
```

Comment:
We have it already in the env, just use it. No need to do something around the input.

See: https://github.com/aws-observability/aws-application-signals-test-framework/blob/main/.github/workflows/java-eks-e2e-test.yml#L232

Reply:
Also, is this variable meant to be a secret? Can we confirm whether it is a risk to have this appear here?

Reply:
I have removed it and used RDS_MYSQL_CLUSTER_SECRETS_USERNAME instead